src/__tests__/gatsby-node.tests.js

```diff
@@ -524,3 +525,267 @@ test('fetchPloneBreadcrumbs returns Gatsby Node for Breadcrumbs', async () => {
+      _type: 'Folder',
+    },
+  ],
+  items_total: 6,
+  tiles: {},
+  tiles_layout: {},
+  title: 'Plone site',
+  _path: '/',
+  _components: {
+    actions: { _path: '/', _id: 'http://localhost:8080/Plone/@actions' },
+    breadcrumbs: {
+      items: [],
+      _path: '/',
+      _id: 'http://localhost:8080/Plone/@breadcrumbs',
+    },
+    navigation: {
+      items: [Array],
+      _path: '/',
+      _id: 'http://localhost:8080/Plone/@navigation',
+    },
+  },
+  _id: 'Plone',
+  _type: 'Plone Site',
+  _parent: {},
+  id: 'http://localhost:8080/Plone',
+  internal: {
+    contentDigest: '9343ca3cbe120f91bb6d7de577ca2872',
+    mediaType: 'text/html',
+    type: 'PloneSite',
+  },
+  _backlinks: [''],
+};
+
+const BreadcrumbsNodeGenerator = {
+  items: [],
+  _path: '/',
+  _id: 'http://localhost:8080/Plone/@breadcrumbs',
+  id: 'http://localhost:8080/Plone/@breadcrumbs',
+  internal: {
+    contentDigest: '42e098df7c173baeefe17561865da7a4',
+    mediaType: 'application/json',
+    type: 'PloneBreadcrumbs',
+  },
+};
+
+const NavigationNodeGenerator = {
+  items: [
+    {
+      description: '',
+      title: 'Home',
+      _path: '/',
+      _id: 'http://localhost:8080/Plone',
+    },
+    {
+      description: '',
+      title: 'gatsby-source-plone',
+      _path: '/index/',
+      _id: 'http://localhost:8080/Plone/index',
+    },
+    {
+      description: 'News on gatsby-source-plone development',
+      title: 'News',
+      _path: '/news/',
+      _id: 'http://localhost:8080/Plone/news',
+    },
+    {
+      description:
+        'A folder with different standard content types that Plone supports out of the box',
+      title: 'Examples',
+      _path: '/examples/',
+      _id: 'http://localhost:8080/Plone/examples',
+    },
+    {
+      description: '',
+      title: 'Tutorial',
+      _path: '/tutorial/',
+      _id: 'http://localhost:8080/Plone/tutorial',
+    },
+    {
+      description: 'Reference documentation',
+      title: 'Docs',
+      _path: '/reference/',
+      _id: 'http://localhost:8080/Plone/reference',
+    },
+    {
+      description: 'UpperFolder',
+      title: 'This is upperFolder',
+      _path: '/this-is-upperfolder/',
+      _id: 'http://localhost:8080/Plone/this-is-upperfolder',
+    },
+  ],
+  _path: '/',
+  _id: 'http://localhost:8080/Plone/@navigation',
+  id: 'http://localhost:8080/Plone/@navigation',
+  internal: {
+    contentDigest: '5c82b5b32c846fb5b65a17bdfe9238e8',
+    mediaType: 'application/json',
+    type: 'PloneNavigation',
+  },
+};
+
+const mockPloneNodeGeneratorget = {
+  get: async (url, headers, params) => {
+    return {
+      data: mockDataPloneNodeGenerator,
+    };
+  },
+};
+let count = 0;
+let expansions;
+test('fetchPloneNodeGenerator return all created node after fetching data form Plone CMS', async () => {
+  for await (const node of ploneNodeGenerator(
+    generatorid,
+    '',
+    generatorbaseUrl,
+    expansions,
```

Comment:
Would just `[]` be enough instead of defining and using `expansions` variable. Or maybe `[]  // expansions` if we want to be more verbose about which argument we are passing.

Reply:
ok!